cmd/solution/fork.go

```diff
@@ -215,7 +215,6 @@ func forkFromDisk(sourceDir string, fileSystem afero.Fs, solutionName string, st
 	} else {
 		solution.Manifest.Name = solutionName
 	}
-	solution.Manifest.SolutionVersion = "1.0.0" // reset version
```

Comment:
To preserve the prior behavior and the use case of forking examples (where the version reset is the right action), I would recommend adding a flag, e.g., `--preserve-solution-version` that enables the new behavior explicitly, leaving the default to reset the version (as introduced in https://github.com/cisco-open/fsoc/pull/342/commits/ef9a08d8a1ec81865b8dae47bcd9e61294a3dd8c) 

Also, while the fsoc-supported pseudo-isolation is now deprecated, it is still available if needed for specific uses. It works in a way that is similar to fork, except it is done transparently on `solution push` and you have control (and the obligation) to identify which instances of the original solution name need to be replaced. The latter is sometimes helpful when the solution name and attribute name element are the same.